Makefile

```diff
@@ -1,7 +1,7 @@
-DOCKER_IMAGE_VERSION=0.0.1
+DOCKER_IMAGE_VERSION=2
```

Comment:
Using official tag naming to avoid confusion and let people know what registry version is used in the image.